source/MonthlyOverview/MonthlyOverview.html

```diff
@@ -11,7 +11,12 @@
         <div id="header">
             <h1 id="year">
                 <h1 id="index">
-                    <a href="../YearlyOverview/YearlyOverview.html" id="back">
+                    <a
+                        href="../YearlyOverview/YearlyOverview.html"
+                        style="text-decoration: none"
+                        id="back"
```

Comment:
You may need to pull from main as I think someone maybe Hunter, changed the code so there isn't CSS with the HTML which you branch still has